src/Reflection/ClassReflection.php

```diff
@@ -113,65 +108,55 @@ public function getMethod($name)
     }
 
     /**
-     * Get reflection objects of all methods
+     * {@inheritDoc}
      *
      * @param  int $filter
-     * @return MethodReflection[]
+     * @return list<MethodReflection>
```

Comment:
I do prefer using `list<string>` syntax over `string[]`, and there are other places this could be updated in the code, in this PR. But I'll leave that to you to decide whether it's worth it.

Reply:
For now, I limited myself to reducing `psalm-baseline.xml` size by a bit.

Indeed, I prefer using `list` and explicit key declarations wherever I have an overview of things to come, but for now, I mostly wanted to reduce the size of this massive minefield :D